modules/install/pages/server-processes.adoc

```diff
@@ -142,3 +142,95 @@ Instance count is configured in UI.
 | Analytics
 | _/opt/couchbase/bin/_
 |===
+
+== Windows
+
+The following table lists the Couchbase processes that run on the Windows platforms.
+
+.Couchbase Processes on Linux
```

Comment:
I see a few processes missing. This is from our list. Would be worth while to check with latest CB version.

WIN_PROCESSES_SPAWNED = ["backup.exe", "cbas.exe", "cbft.exe", "cbq-engine.exe",
                         "erl.exe", "eventing-producer.exe", "goxdcr.exe",
                         "indexer.exe", "java.exe", "memcached.exe",
                         "projector.exe", "prometheus.exe", "saslauthd-port.exe",
                         "eventing-consumer.exe"]

Reply:
Where does that list come from?